DFC.ServiceTaxonomy.PageLocation/Migrations.cs

```diff
@@ -25,7 +26,7 @@ public int Create()
 
         public int UpdateFrom1()
```

Comment:
Can we have a better naming convention for migrations that describe the change? If not, when we get loads it will be messy very quickly.